iroha/src/block_sync.rs

```diff
@@ -102,15 +101,16 @@ impl BlockSynchronizer {
                     .read()
                     .await
                     .network_topology_current_or_genesis(&block.clone().into());
-                if block.header().number_of_view_changes < self.n_topology_shifts_before_reshuffle {
+                if block.header().number_of_view_changes <= self.n_topology_shifts_before_reshuffle
```

Comment:
Why it should be `<=` instead of `<` here?

Reply:
There was a bug in block sync, basically there was a difference exactly in this sign because in sumeragi +1 change was added later and here it was already encoded in a block.

Reply:
Oh, i see. Great then!